site/class-1-play/japgroevemaker.svg

```diff
@@ -0,0 +1,52 @@
+<svg xmlns="http://www.w3.org/2000/svg">
```

Comment:
Your SVG is late (and broken). Could you remove it? you can do that by opening the file on your branch on your fork, and clicking on the trash icon in the top-right corner.